### PR TITLE
feat: discordBadge addition

### PR DESCRIPTION
**Pull request by Ruchika30**
This commit adds a  discordBadge which will redirect the user to the platform for discussion

Closes #

## Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Ruchika30** [comment]
@dbartholomae I am getting "Type '{ inviteLink: string; serverId: string; }' is not assignable to type 'IntrinsicAttributes & Readonly<BadgeProps>'. Property 'inviteLink' does not exist on type 'IntrinsicAttributes & Readonly<BadgeProps>'." 

 When I added DiscordBadge in Readme.md.tsx.  Though I am not using BadgeComponent in my DoscordBadge. Kindly assist

**Comment by dbartholomae** [comment]
Thanks! The code looks good. Could you squash the commits into one? Each commit will create a line in the Changelog, so a single feat commit should be enough.

File: src/components/badges/DiscordBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* @jsx MD */
+import MD, { Component } from "jsx-md";
+import { Badge } from "../Badge";
+
+/** Show a discord badge  on GitHub */
```
Comment (dbartholomae) [inline src/components/badges/DiscordBadge.tsx:5]: This line needs to be directly in front of the DiscordBadge line (before line 13). And there is a double space in it.

**Comment by dbartholomae** [comment]
One more thing I found was that the documentation comment is not placed corrcetly

**Comment by Ruchika30** [comment]
> Thanks! The code looks good. Could you squash the commits into one? Each commit will create a line in the Changelog, so a single feat commit should be enough.

File: src/components/badges/DiscordBadge.tsx (hunk 1 above)
Comment (Ruchika30) [reply src/components/badges/DiscordBadge.tsx]: done

**Comment by Ruchika30** [comment]
> Thanks! The code looks good. Could you squash the commits into one? Each commit will create a line in the Changelog, so a single feat commit should be enough.

Done in new PR

**Comment by Ruchika30** [comment]
> One more thing I found was that the documentation comment is not placed corrcetly

Did u mean the one you pointed in DiscordBadge.tsx file?

**Comment by dbartholomae** [comment]
Superseded by #21

**Pull request closed without merging**